src/index/TextScoring.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Felix Meisen (fesemeisen@outlook.de)
+
+#pragma once
+
+#include "index/Index.h"
+#include "parser/WordsAndDocsFileParser.h"
+
+class ScoreData {
+ public:
+  ScoreData() : scoringMetric_(TextScoringMetric::COUNT), b_(0.75), k_(1.75){};
+
+  ScoreData(LocaleManager localeManager)
+      : scoringMetric_(TextScoringMetric::COUNT),
+        b_(0.75),
+        k_(1.75),
```

Comment:
Use default initializers for the class members I am sure sonarcloud also told you this.